integrated_channels/cornerstone/transmitters/content_metadata.py

```diff
@@ -31,14 +31,14 @@ def transmit(self, payload, **kwargs):
             payload (OrderedDict): course metadata dictionary corresponding to the result of the content metadata
             exporter's export.
         """
-        items_to_create, items_to_update, items_to_delete, transmission_map, items_catalog_updated_times = \
-            self._partition_items(payload)
+        items_to_create, items_to_update, items_to_delete, transmission_map, items_catalog_updated_times, \
```

Comment:
we had circled the idea of embedding the catalog uuid in the items_to_create itself but it seemed safer to leave those untouched for the time being